stages/org.osbuild.rhsm.facts

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+"""
+This stage allows storing artifact-properties of the Red Hat Subscription
+Manager (rhsm-facts) in the built image.
+"""
+
+import json
+import os
+import sys
+
+import osbuild.api
+
+
+SCHEMA = r"""
+"properties": {
+  "facts": {
+    "type": "object",
+    "description": "Map of Red Hat Subscription Manager facts.",
+    "additionalProperties": true
+  }
+}
+"""
+
+
+def main(tree, options):
+    path = os.path.join(tree, "usr/share/osbuild/self")
```

Comment:
I think @teg was not a fan `osbuild/self`. Maybe we just put it in  `/usr/share/osbuild/rhsm.facts/`?

Reply:
Lets maybe discuss this separately (slack?) There's now 3 threads about that in here, and all have at one point yielded and agreed on "self" ;)